content/docs/connect/choose-connection.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Choosing your connection type
+subtitle: Rules of thumb for choosing one connection type over another
+enableTableOfContents: true
+---
+
+When setting up your application’s connection to your Neon Postgres database, you need to make two main choices: the type of driver and the type of connection. This flowchart will guide you through these options.
+
+## Choosing your connection type: flowchart
+
+![choose your connection type](/docs/connect/choose_connection.png)
```

Comment:
Not sure about "Is your pool size limited by the default 64 connections?" By default, everyone has a PgBouncer setting of default_pool_size=64. So everyone's pool is limited to 64 unless that setting is changed. 

Maybe the question here is: 

Will an individual Postgres role require more than 64 concurrent connections to a database? 

If the answer is "yes"

"Contact Support to increase your PgBouncer default_pool_size setting."

Reply:
Modify "unpooled" in the rest of the docs before finishing this PR, pending confirmation.

Reply:
Modified the flowchart. Flowchart seems busy, but maybe PixelPoint can make it feel simpler.